pkg/providers/tinkerbell/pbnj/client.go

```diff
@@ -0,0 +1,65 @@
+package pbnj
+
+import (
+	"context"
+	"os"
+
+	"github.com/tinkerbell/cluster-api-provider-tinkerbell/pbnj/client"
+	"github.com/tinkerbell/pbnj/api/v1"
+)
+
+const (
+	PbnjGrpcAuth = "PBNJ_GRPC_AUTHORITY"
```

Comment:
Do we expect package consumers to need to use this at all? Seems unnecessary if they explicitly pass in an authorization parameter to the PBnJ client on l. 26.

Reply:
@panktishah26 does this need to be exported?